packages/api/src/controllers/region.js

```diff
@@ -6,20 +6,22 @@ const defaultScore = 1;
 
 const app = Router();
 
-function flatRegions(regions = []) {
+function flatRegions(regions = [], halfRegionOrchestratorsUntrusted = false) {
   return regions.flatMap((reg) =>
-    reg.orchestrators.map((orch) => ({
-      score: defaultScore,
+    reg.orchestrators.map((orch, i) => ({
+      score: halfRegionOrchestratorsUntrusted && i % 2 == 0 ? 0 : defaultScore,
       region: reg.region,
       ...orch,
     }))
   );
 }
 
-export async function regionsGetter() {
-  const [regions, cursor] = await db.region.find({}, { limit: 100 });
+export function regionsGetter(halfRegionOrchestratorsUntrusted = false) {
+  return async function regionsGetter2() {
```

Comment:
@livepeer/video-user I feel like this is probably not the optimal way, so would appreciate any pointers on how I should be doing it!

Reply:
I think this works fine! One syntax sugar you could use to make this look a little better would be a lambda function, replacing `async function regionsGetter2() {` with `async () => {`. Mostly a stylistic choice anyway.

Another alternative, to avoid this helper function that returns a function (which I find a little confusing sometimes) is to curry the new boolean variable on the call site where you need this function with zero arguments instead. That is, instead of returning a function here you could just make this function receive the boolean and return the regions, and in `app-router` you do:
`orchestratorGetters.push(() => regionsGetter(halfUntrusted...))`.

This also fixes the issue of using the return value of this function (another function) when we actually want the final list of regions.

This is a totally optional/stylistic change anyway, only raising the possibility since you were a little concerned about the current version here.